src/time.rs

```diff
@@ -13,13 +16,8 @@
 /// # Safety
 /// * Causes a GP fault if the TSD flag in register CR4 is set and the CPL
 ///   is greater than 0.
-#[allow(unused_mut)]
 pub unsafe fn rdtsc() -> u64 {
-    let mut low: u32;
-    let mut high: u32;
-
-    asm!("rdtsc" : "={eax}" (low), "={edx}" (high));
-    ((high as u64) << 32) | (low as u64)
+    mem::transmute(_rdtsc())
```

Comment:
Isn't it *much* better to use `as u64` for this sort of thing?

Reply:
It is... It's fixed on master now thanks for raising this concern.